apix-docker/231/docker-compose.yml

```diff
@@ -33,16 +36,16 @@ services:
     restart: unless-stopped
 
   activemq:
-    image: alfresco/alfresco-activemq:5.16.1
+    image: alfresco/alfresco-activemq:5.17.1-jre11-rockylinux8
```

Comment:
the whole project moved to java 17, why not use it here as well? `5.17.6-jre17-rockylinux8`

Reply:
Good catch!